spec/lib/runner_spec.rb

```diff
@@ -369,6 +443,10 @@ def handle_valid_record(record, data_type)
       @runner = test_runner('bot-with-transformer')
     end
 
+    after do
+      cleanup_test_runner(@runner)
+    end
```

Comment:
It's a bit repetitive isn't it! I guess you can't simply put at top of file because there are some specs that don't have a `@runner`. But maybe those specs could be tweaked so they do..?

Reply:
ok yes. I have put made all specs use `@runner` consistently, and then put this in an after block at the top instead. ✅